tests/e2e/vsan_test.go

```diff
@@ -77,12 +82,24 @@ func (s *VsanTestSuite) TearDownTest(c *C) {
 
 var _ = Suite(&VsanTestSuite{})
 
+func (s *VsanTestSuite) newCName() string {
+	cname := inputparams.GetUniqueContainerName("vsan_test")
+	s.containerList = append(s.containerList, cname)
+	return cname
+}
+
 // Steps:
 // 1. Create a valid vsan policy
 // 2. Volume creation with valid policy should pass
 // 3. Valid volume should be accessible
+// 4. Start a container to mount the volume, and write message to the volume
+// 5. Start another container to mount the volume, and read message from volume and verify the
```

Comment:
I still don't follow: the test logic shows that we are starting a container to mount the volume and write data to the volume, which is step 4. But where is the logic of "**start another container to mount the volume**" as described in step 5?

I guess step 5 should be just: "Read message from volume and verify the message..."